swabbie-aws/src/main/kotlin/com/netflix/spinnaker/config/EddaConfiguration.kt

```diff
@@ -100,26 +107,43 @@ open class EddaConfiguration {
     return listOf(accountEddas, endpointEddas).flatten()
   }
 
+  @Bean
+  open fun eddaEndpointCache(eddaEndpointsService: EddaEndpointsService): EddaEndpointCache {
+    return EddaEndpointCache(eddaEndpointsService)
```

Comment:
In a future step, these will default to just vanilla aws unless `edda.enabled: true` is set, right?

Reply:
Yeah. I am planning to rev this up once I put in aws stuff.